mantle/cmd/kola/testiso.go

```diff
@@ -675,59 +675,85 @@ func awaitCompletion(ctx context.Context, inst *platform.QemuInstance, outdir st
 					err = platform.ErrInitramfsEmergency
 				}
 			}
+		}()
+		go func() {
+			err := inst.Wait()
+			// only one Wait() gets process data, so also manually check for signal
+			plog.Debugf("qemu exited err=%v", err)
+			if err == nil && inst.Signaled() {
+				err = errors.New("process killed")
+			}
 			if err != nil {
-				errchan <- err
+				errchan <- errors.Wrapf(err, "QEMU unexpectedly exited while awaiting completion")
 			}
+			time.Sleep(1 * time.Minute)
+			errchan <- fmt.Errorf("QEMU exited; timed out waiting for completion")
 		}()
-	}
```

Comment:
are these changes desired? maybe some accidental delete?

Reply:
That's correct, I was working on some debugging there, this is a leftover, thanks.